lib/domain/models/contributor.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:equatable/equatable.dart';
+import 'package:meta/meta.dart';
+
+// ignore: comment_references
+/// One of the [Contributor]'s in the [Collection]
+@immutable
+class Contributor extends Equatable {
+  const Contributor({
+    required this.id,
+    required this.name,
+    required this.url,
+    required this.imageUrl,
+  });
+
+  final String id;
+
+  /// This [Contributor] github user with the [@]
+  final String name;
+
+  /// This [Contributor] image url
+  final String imageUrl;
+
+  /// URL that links to this particular [Contributor]
```

Comment:
👍

Reply:
Actually, I changed this statement to something similar:

`/// An external url - chosen by the collection contributor - which is linked within the 'SingleContributorButton'`

Because the link is created at the onTap event within the SingleContributorButton.

Reply:
@ggirotto

Reply:
The problem of referencing `SingleContributorButton` inside the `Contributor` entity is that if this component is updated for some reason (renamed, removed, whatever), this comment will become outdated and will not be updated until someone notices it.

I suggest something agnostic from the interface, like
`An external url which references a website from the user choice to be used in self-promotion contexts`

Or something like that

Reply:
Got it. 👍